modules/op-release-notes-1-8.adoc

```diff
@@ -123,13 +123,30 @@ Because this feature is available by default, you no longer need to set the `pip
 === Operator
 
 // [Hub]
-* With this update, as an administrator, you can configure your local {tekton-hub} instance to use a custom database rather than the default database.
+* With this update, as an administrator, you can configure your local {tekton-hub} instance to use a custom PostgreSQL 13 database rather than the default database. To do so, create a `tekton-hub-db` `Secret` instance. For example:
```

Comment:
Overall it looks good,
But instead of updating this point here, can we add this point below under Tekton Hub Section

Reply:
For ref I have added  a comment under Tekton Hub section https://github.com/openshift/openshift-docs/pull/49137/files#r951366928 in this pr